docs/specification.md

```diff
@@ -49,13 +51,12 @@ The "transfer hook" approach is recommended, but may be infeasible due to gas co
 3. We will not implement any `owner_hook`.
 The contract is usable without them because we stick to `Optional_owner_hook`.
```

Comment:
I didn't understand this part. Isn't owner hook interface part of the receivers (contracts) of the USDC token? 
AFAIU optional owner hook just means that a transfer won't fail if the sender/reciever contracts does not implement an owner hook.

Reply:
> AFAIU optional owner hook just means that a transfer won't fail if the sender/reciever contracts does not implement an owner hook.

Yes. I. e. we let each user decide whether they want to implement a hook or not. So if Alice wants to do something when funds are transferred to her, she will do it. If not, she won't do it and she can use a regular `tz*` address. That's what I mean by saying that the contract is usable. So even if we don't implement any owner_hook, anyone can take this contract and use it. If made owner hook mandatory, the contract wouldn't be usable without them.

Reply:
I think the confusing part is "We will not implement any `owner_hook`.". It would have made sense to me if it said "We will not require any owner_hook". But I get what you mean.